bentoml/_internal/models/store.py

```diff
@@ -187,6 +186,24 @@ def export_model(self, name: str):
     def import_model(self, name: str):
```

Comment:
on a side note, similar to the bento_store, the `delete_model` above will need to search for the `latest` based on file timestamp and recreate the symlink iff the model being deleted is the current "latest"